packages/devtools_app/lib/src/framework/scaffold/about_dialog.dart

```diff
@@ -144,3 +144,29 @@ class OpenAboutAction extends ScaffoldAction {
         },
       );
 }
+
+/// Since [DevToolsAboutDialog] is not actually an [AboutDialog], there is no
+/// built-in way to add a 'View Licenses' button.
+/// So, adding a custom [DialogTextButton] to view licenses.
+/// Since this action is very specific to just the [DevToolsAboutDialog],
+/// providing implementation in about_dialog.dart and not in
+/// dialogs.dart which contains the definition of [DevToolsDialog].
+/// TODO(mossmana): We may want to consider refactoring [DevToolsAboutDialog] to be an [AboutDialog].
+/// https://api.flutter.dev/flutter/material/AboutDialog-class.html
```

Comment:
this should use // instead of /// since it is not dart doc for the class, but rather comments to explain why we used this approach.